src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java

```diff
@@ -12,8 +12,8 @@
   ICHHQW(ActionHandler.PRINTER), // Census household questionnaire for Wales
   ICHHQN(ActionHandler.PRINTER), // Census household questionnaire for NI
 
-  // Fieldwork reminders
-  FF2QE(ActionHandler.FIELD), // Fieldwork follow up F2 questionnaire for England
+  // generic actionType for use in Fieldwork followup actionplans, tranches
```

Comment:
Tiny suggestion to avoid confusion:
```suggestion
  // Generic actionType for use in Fieldwork followup action rules, tranches
```